src/pages/Dashboard/components/PeopleList.jsx

```diff
@@ -1,15 +1,16 @@
-import PeopleListItem from './PeopleListItem'
+/* eslint-disable react/prop-types */
+import PeopleListItem from "./PeopleListItem";
 
 function PeopleList(props) {
```

Comment:
you can merge the peoplelist + listitem in one since both are quite small / concise